codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -108,8 +108,15 @@ public void channelHandlerContext(ChannelHandlerContext ctx) {
         this.ctx = ctx;
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * This method must be called from the same event loop context as the {@link ChannelHandlerContext}
```

Comment:
@Scottmitch agreed, we would definitely want some consistency with when `@NotThreadSafe` is used and when it's missing.  Perhaps we could just be consistent within HTTP/2 for now and then tackling the rest of Netty in a follow-up PR? I suspect it would be worthwhile documentation for everyone using Netty.

@normanmaurer any thoughts on using `@NotThreadSafe` in Netty?